demo/index.html

```diff
@@ -316,7 +302,135 @@ <h1>Symphony Electron API Demo</h1>
             document.location.reload();
         });
 
-        window.addEventListener('message', (event) => console.log(event.data));
+        /**
+         * Activity Detection
+         */
+
+        const activityDetection = document.getElementById('activity-detection');
+        const activityStatus = document.getElementById('activity-status');
+        activityDetection.addEventListener('click', () => {
+            if (window.ssf) {
+                window.ssf.registerActivityDetection(10000, activityDetection);
+            } else {
+                postMessage(apiCmds.registerActivityDetection, 5000);
```

Comment:
why is the timer different here from ssf?

Reply:
Both of them works. In `ssf` it is the same as the logic that changes the colour.